.pre-commit-config.yaml

```
@@ -32,15 +32,15 @@ repos:
           - "@typescript-eslint/eslint-plugin@5.42.1"
           - "eslint-plugin-prefer-arrow@1.2.3"
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v2.7.1
+    rev: v3.0.0-alpha.9-for-vscode
```

```suggestion
    rev: v2.7.1
```
Please do not blindly execute scripts, but also read their output:

> Updating pre-commit hooks...
> Note: Please downgrade any alpha versions manually

The problem is that `pre-commit autoupdate` always takes the latest version, even if they're alpha or beta versions... I did not find a way around it unfortunately...

---

I'm sorry!